### PR TITLE
test: cypress update

### PR DESCRIPTION
**Pull request by JeevaRamu0104**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [x] CI/CD

## Description
Added cypress test for the Login and signUp module.

## How did you test it?
https://github.com/juspay/hyperswitch-control-center/actions/runs/10938944305/job/30368147456?pr=1420

## Where to test it?

- [x] INTEG
- [x] SANDBOX
- [ ] PROD

## Checklist

- [ ] I ran `npm run re:build`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/components/Button.res [hunk 1]
```diff
@@ -293,6 +293,7 @@ let make = (
   ~toolTipPosition=ToolTip.Top,
   ~dataTestId="",
 ) => {
+  Js.log(dataTestId)
```
Comment (gitanjli525) [inline src/components/Button.res:296]: is this log intentional?

File: package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "re:format": "rescript format -all",
     "test:start": "node dist/server/server.js",
     "postinstall": "git config core.hooksPath .githooks && chmod +x .githooks/commit-msg",
-    "cy:open": "sh cypress/start_hyperswitch.sh && cypress open",
+    "cy:open": "sh cypress/start_hyperswitch.sh && cypress run",
     "cy:run": "sh cypress/start_hyperswitch.sh && cypress run",
```
Comment (gitanjli525) [inline package.json:22]: cy:open and cy:run both will trigger the same command, are both required ?

Comment (JeevaRamu0104) [reply package.json]: updated

File: src/components/Button.res (hunk 1 above)
Comment (JeevaRamu0104) [reply src/components/Button.res]: no remove the log

**Review by gitanjli525 — APPROVED** [review]

**Pull request merged**